### ISSUE TITLE
Relative hyperlinks are not rewritten to the assets folder

### ISSUE THREAD

**Issue by MCTaylor17** [BCIT-LTC/dewordify#17]
Written on behalf of @kphunter 

Currently, dewordify rewrites hyperlinks that start with `file:///` to point at the assets folder.  Relative files are likely to come this way as they are rewritten by MS Word to create an absolute path from the persons computer.  Unfortunately, this only seems to be the case with "Save-As".  If a user never performs a save-as after adding a relative hyperlink path like this:

![image](https://user-images.githubusercontent.com/7216377/28391560-5f3295cc-6c93-11e7-97c1-e4ed700122a2.png)

Then the output from dewordify will not point at the assets folder.  Instead the url will look like this:

```html
<a href="something.pdf">something</a>
```

Ideally, the url would appear like this:

```html
<a href="assets/something.pdf">something</a>
```

**Comment by MCTaylor17** [comment]
Solved by 85818c1

**Comment by kphunter** [comment]
@MCTaylor17 Much appreciated! Thanks.
